content/docs/setup/kubernetes/istio-cni-install/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Istio CNI Installation
+description: Instructions for installing and using Istio with the Istio CNI plugin to allow Istio users to deploy services via lower privileged RBAC.  The Istio CNI plugin removes the requirement for the privileged, `NET_ADMIN` initialization container in the users' pods in the mesh.
```

Comment:
Please keep the description to a single sentence. I'd just nuke the second sentence.

Reply:
done